### ISSUE TITLE
bash completion file is missing from the deb package

### ISSUE THREAD

**Issue by letompouce** [sharkdp/bat#1993]
**Describe the bug you encountered:**

bash compeltion file is missing from the .deb package.

**What did you expect to happen instead?**

bash completion file is provided in the .deb package and is installed.

**How did you install `bat`?**

On my systems, `wget` the `.deb` file then `dpkg -i` it. Reproducible in a fresh Debian container:

```
$ docker run --rm -it debian
root@fe53f0e22d48:/# cat /etc/debian_version 
11.2
root@fe53f0e22d48:/# apt-get update &> /dev/null && apt-get install -y wget bash-completion &> /dev/null    
root@fe53f0e22d48:/# wget -q https://github.com/sharkdp/bat/releases/download/v0.18.3/bat_0.18.3_amd64.deb && dpkg -i *.deb
Selecting previously unselected package bat.
(Reading database ... 7837 files and directories currently installed.)
Preparing to unpack bat_0.18.3_amd64.deb ...
Unpacking bat (0.18.3) ...
Setting up bat (0.18.3) ...
root@fe53f0e22d48:/# dpkg -L bat
/.
/usr
/usr/bin
/usr/bin/bat
/usr/share
/usr/share/doc
/usr/share/doc/bat
/usr/share/doc/bat/LICENSE-APACHE
/usr/share/doc/bat/LICENSE-MIT
/usr/share/doc/bat/README.md
/usr/share/doc/bat/changelog.gz
/usr/share/doc/bat/copyright
/usr/share/fish
/usr/share/fish/vendor_completions.d
/usr/share/fish/vendor_completions.d/bat.fish
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/bat.1.gz
/usr/share/zsh
/usr/share/zsh/vendor-completions
/usr/share/zsh/vendor-completions/_bat
root@fe53f0e22d48:/# find /usr/ -iname '*bat*' | fgrep comp
/usr/share/fish/vendor_completions.d/bat.fish
/usr/share/zsh/vendor-completions/_bat
root@fe53f0e22d48:/#
```

---

**bat version and environment**

Tested with the latest 0.18.3, but also happens with the 0.18.2 version the bash completion was added in (issues #1678 & #1733 / 47283f2) .

```
# bat -<TAB><TAB><TAB><TAB><TAB><TAB><TAB><TAB>^C
# bat -V
bat 0.18.3
```

**Comment by sharkdp** [comment]
https://github.com/sharkdp/bat/pull/1848 (which fixes https://github.com/sharkdp/bat/issues/1733) has been merged after bat 0.18.3 has been released. So this will only be available in the next release.

**Comment by sharkdp** [comment]
closing for now.
